guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -12,10 +12,10 @@ include::snip_firewalld.adoc[]
 +
 ----
 ifdef::katello,satellite,orcharhino[]
-# dnf install postgresql-server postgresql-evr
+# dnf install postgresql-server postgresql-evr postgresql-contrib
 endif::[]
 ifndef::katello,satellite,orcharhino[]
-# dnf install postgresql-server
+# dnf install postgresql-server postgresql-contrib
```

Comment:
It's only needed for Pulp
```suggestion
# dnf install postgresql-server
```

Reply:
Noted. Should I raise the PR (with the same changes) for `3.7` only? And close this PR?

Reply:
No, everything always goes through master and is then cherry picked. I wanted to make it clear it should not be picked to 3.6 and 3.5.

Reply:
Alright, I have unchecked those versions :+1:

Reply:
Addressed :+1: